### PR TITLE
lte: restore lens control plane

### PR DESCRIPTION
**Pull request by joemfb**
This PR is the dual of urbit/urbit#1277. %rver implements the new %request-local task (thanks @ixv!), and there's an incredibly basic, one-request-at-a-time lens app. Just enough to get the CI and pill staging back to a working state (on urbit/urbit).

### CODE REVIEW

File: sys/vane/rver.hoon [hunk 1]
```diff
@@ -675,6 +675,31 @@
...
+      [act [& secure address request] ~ 0]
+    ::
+    =.  connections.state
+      (~(put by connections.state) duct connection)
+    ::
+    :_  state
+    :_  ~
+    :^  duct  %pass  /run-app/[app.act]
+    ^-  note
+    :^  %g  %deal  [our our]
```
Comment (eglaysher) [inline sys/vane/rver.hoon]: I worry a bit about how this appears to be hardcoding everything to the %lens app, but I suppose that's fine and maintaining backwards compatibility.

Comment (joemfb) [reply sys/vane/rver.hoon]: Right - this PR is just restoring the old lens functionality. I think the correct approach would be to make lens (and herb) make and handle generic http requests, using dynamically-bound routes, generalized across local and remote servers. Then the only difference between `%request` and `%request-local` would be that the latter is pre-authorized. But all that will take some work.

**Comment by ohAitch** [comment]
It seems a lot more robust to use the standard eyre app authentication
mechanism, perhaps with a lens token written out into .urb for shell
access. I think a lot of the lens port separation was there to quarantine
code shoehorning "%poke with empty ack, console %peer with several spurious
responses" into a single JSON rpc call-response, which looks like it's now
directly supported / entirely in userspace?

**Comment by joemfb** [comment]
Full stateful http-request handling is now in userspace (with lighter-than-eyre), Lens/herb/urb.py should be changed to work that way, and to support the standard cookie-based authentication. But I still think it's worth keeping around the lens server as a automatically-authorized local-only channel -- it's handy for scripting, CI, etc.

**Pull request merged**